src/async/tasks.md

```diff
@@ -14,34 +14,27 @@ use tokio::net::TcpListener;
 #[tokio::main]
 async fn main() -> io::Result<()> {
     let listener = TcpListener::bind("127.0.0.1:6142").await?;
```

Comment:
If we want to demonstrate ephemeral ports, this should be `0` with a comment indicating what that means.

```suggestion
    let listener = TcpListener::bind("127.0.0.1:0").await?;
```